lib/liveValidation/liveValidator.ts

```diff
@@ -174,6 +177,7 @@ export class LiveValidator {
     this.options = ops as LiveValidatorOptions;
     this.logging(`Creating livevalidator with options:${JSON.stringify(this.options)}`);
     this.operationSearcher = new OperationSearcher(this.logging);
+    this.operationSpecMapper = new Map<string, string[]>();
```

Comment:
How about moving this information to trafficValidation class as it's not useful here?